packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec

```diff
@@ -1,34 +1,40 @@
+# template: hammer_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
 %global gem_name hammer_cli_foreman_remote_execution
-%global confdir hammer
+%global plugin_name foreman_remote_execution
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
 
-Summary: Foreman Remote Execution commands for Hammer CLI
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.1
+Version: 0.1.2
 Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
+Summary: CLI for the Foreman remote execution plugin
+Group: Development/Languages
+License: GPL-3.0-or-later
```

Comment:
This is the SPDX license tag, not the RPM one.

Reply:
tell `licenses.json` (like I did now ;P)